README.md

```diff
@@ -39,20 +48,25 @@ Run with profile from [memprof](https://github.com/mrc-ide/memprof)
 
 ``` r
 source("threemc_fit.R")
-mem <- memprof::with_monitor(threemc_fit())
+# load shell dataset
+shell_dat <- readr::read_csv("data/shell_data_lso.csv.gz")
 #> Rows: 13200 Columns: 15
-#> ── Column specification ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+#> ── Column specification ────────────────────────────────────────────────────────
 #> Delimiter: ","
 #> chr  (2): area_id, area_name
-#> dbl (13): area_level, space, year, circ_age, time, age, population, N, obs_mmc, obs_tmc, obs_mc, cens, icens
+#> dbl (13): area_level, space, year, circ_age, time, age, population, N, obs_m...
 #> 
 #> ℹ Use `spec()` to retrieve the full column specification for this data.
 #> ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.
+# load shapefiles
+areas <- sf::read_sf("data/areas.geojson")
+# load text from C++ mod into R
+mod <- readLines("src/threemc.cpp")
+mem <- memprof::with_monitor(threemc_fit(shell_dat, areas, mod))
 #> area_lev arg missing, taken as maximum area level in shell dataset
-#> Warning: as(<matrix>, "dgTMatrix") is deprecated since Matrix 1.5-0; do as(as(as(., "dMatrix"), "generalMatrix"), "TsparseMatrix") instead
+#> Warning: as(<matrix>, "dgTMatrix") is deprecated since Matrix 1.5-0; do
+#> as(as(as(., "dMatrix"), "generalMatrix"), "TsparseMatrix") instead
 #> rw_order = NULL, AR 1 temporal prior specified
-#> Note: Using Makevars in /home/rashton/.R/Makevars 
-#> Optimizing tape... Done
 #> iter: 1  value: 30677.38 mgc: 1802.578 ustep: 0.004471122 
```

Comment:
Do we have a way we can get the model to run silently? I think all this not that interesting for what we are asking for help with

Reply:
Yep, can have `MakeADFun(..., silent = TRUE)`, will do